docs/index.html

```diff
@@ -8,6 +8,7 @@
   <meta name="keywords" content="CSS, Animation, Easy as hell">
   <meta name="author" content="linxea">
   <link rel="shortcut icon" type="image/x-icon" href="favicon.ico">
+  <link rel="stylesheet" type="text/css" href="example_animation.css">
```

Comment:
Can we move this to a new index.html? For example `/docs/example/index.html` so it doesn't change the default index.html in `/docs`.

Reply:
ok i just look to this so basically we want the index,html unchanged?

Reply:
But if its needed to link css with index.html so we need to atleast link the stylesheet otherwise wouldnt copying index.html in other folder be redundant

Reply:
Are you planning to add new style to existing page at /docs/index.html which is loaded at `https://linxea.github.io/css-animation/`? 

Or are you planning to add a new css animation example which will be loaded in a whole new page like  `https://linxea.github.io/css-animation/float/`?  You can refer to this PR https://github.com/linxea/css-animation/pull/8 which is a good example.

Reply:
oh thanks @linxea i will just do the changes and send pr again according to PR #8

Reply:
nice! can't wait to see the PR :)